tests/standalone/e2e/ilab_standalone_rhoai_test.go

```diff
@@ -38,6 +38,15 @@ const (
 	STANDALONE_FILE_PATH        = "../../../standalone/standalone.py"
 )
 
+const (
+	SDG_PIPELINE_FILEPATH = "/usr/share/instructlab/sdg/pipelines/agentic"
+)
+
+const (
+	TEST_APP_LABEL       = "ilab-on-ocp-e2e"
+	E2E_TEST_RUN_TIMEOUT = 10 * time.Hour // 10 hours
+)
```

Comment:
nit: can we leave comments on the organization of these const() blocks? atm it looks arbitrary